main/guides/governance/index.md

```diff
@@ -0,0 +1,263 @@
+# Contract Governance
+
+To help build systems with a good balance of decentralization and executive control, the Agoric platform includes, in addition to chain-wide governance included in the Cosmos SDK platform, an `@agoric/governance` package with a flexible archtecture supporting 3 main features:
```

Comment:
Please add something saying the following. It might be in this paragraph, or it might go after the bullet points. Feel free to reword.

The emphasis, throughout the design of the governance library, is on ensuring that clients of a contract have visibility into who can make changes, what changes they can make, and what the current state of the controlled values is. 

I would also add a sentence or a few more words after each of the bullet points. The first one is clear enough on its own, but the latter two phrases probably won't mean much to most readers at this point.

* API governance -- give governance the ability to invoke particular methods called out by the contract's authors
* Offer filters -- give governance the ability to disable the use of some offers/invitations to deal with emergencies.

Reply:
yes, I meant to elaborate the bullets. thanks for the suggestions.

but
> give governance ...

personifies governance in a way that seems awkward. "give the electorate ..." would fit better, but we haven't introduced the term "electorate" yet. hm.